docs/3.0-migration-guide.d/default-curves.md

```diff
@@ -0,0 +1,16 @@
+Weak curves are now disabled by default for X.509 and TLS
```

Comment:
Should probably say "and hashes". Also, I'm not sure it's fair to call them weak (I guess it will still be some time before we see the first collision of SHA-224 or discrete log computation on secp224r1), but I understand this is a one-line summary. Perhaps "weaker"? Or perhaps "Strengthen default set of curves and hashes"?

Reply:
“Weak” is technically correct since we're disabling mechanism with a nominal security strength that's less than a certain threshold, but I agree that it's weird in this context. I'll think about how to word this better.